pages/contact.html

```diff
@@ -26,20 +27,15 @@
                 <div class="ui padded basic segment">
                     <h1 class="ui block header center blue aligned">Contact Details</h1>
                     <div class="ui center aligned basic segment">
-                        <p>Email School:<a href="mailto:info@mountsioncbssecondary.ie" target="_top">info@mountsioncbssecondary.ie</a></p>
-                        <p>Web: mountsioncbssecondary.ie</p>
-                        <p>Telephone Number: 051 377378</p>
-                        <p>Fax Number: 051-376468</p>
-                        <p>School Roll No. 64930i</p>
-                        <p>Address:</p>
-                        <p>Mount Sion CBS Secondary School,</p>
+                        <p><a href="mailto:info@mountsioncbssecondary.ie"><i class="envelope link icon"></a></i>info@mountsioncbssecondary.ie</p> 
+                        <p><a target="_blank" href="http://www.mountsioncbssecondary.ie"><i class="globe icon"></i></a>mountsioncbssecondary.ie</p>
```

Comment:
@evanhearne what do you think about removing this one? Once we publish this site, we'll be removing the old one so we probably don't want to link out to it. wdyt?

Reply:
Yes, @laurafitzgerald that sounds good. I will remove that now.